interfaces.go

```diff
@@ -11,15 +11,20 @@ import (
 	"slices"
 	"strings"
 	"time"
+
+	"github.com/getsentry/sentry-go/attribute"
 )
 
-// eventType is the type of an error event.
+// eventType is the type of error event.
```

Comment:
```suggestion
// eventType is the default event type.
```

Messages are not errors.